src/test/java/edu/kis/powp/jobs2d/events/SelectTestClone2OptionListener.java

```diff
@@ -0,0 +1,20 @@
+package edu.kis.powp.jobs2d.events;
+
+import edu.kis.powp.jobs2d.command.ComplexCommand;
+import edu.kis.powp.jobs2d.command.ComplexCommandFactory;
+import edu.kis.powp.jobs2d.command.DriverCommand;
+import edu.kis.powp.jobs2d.command.manager.CommandManager;
+import edu.kis.powp.jobs2d.features.CommandsFeature;
+
+import java.awt.event.ActionEvent;
+import java.awt.event.ActionListener;
+
+public class SelectTestClone2OptionListener implements ActionListener {
+
+    @Override
+    public void actionPerformed(ActionEvent e) {
+        DriverCommand comandToClone = ComplexCommandFactory.getRectangle();
```

Comment:
Rectangle is not a multiply nested complex command. The test also does not verify that cloned commands are separate instances. It is of course important when cloning a nested complex command commands list. That's way I suggested a unit test.